### DISCUSSION TITLE
Headless SXA Breadcrumb component

### DISCUSSION THREAD

**Discussion by santhosh4184** [Sitecore/jss#1549] [Help]
### Description

Do we have OOTB Breadcrumb component in Sitecore SXA JSS? If it is not available,
what will be a approach to create it? appreciate your help.

**Comment by art-alexeyenko** [comment]
Navigation component with Breadcrumb style is probably what you're looking for @santhosh4184 
https://doc.sitecore.com/xp/en/users/sxa/18/sitecore-experience-accelerator/the-navigation-renderings.html

If it's not - you can use the Sitecore first approach to make your own implementation for the new component:
https://doc.sitecore.com/xp/en/developers/hd/19/sitecore-headless-development/walkthrough--creating-a-new-component-in-a-jss-next-js-app-using-the-sitecore-first-development-workflow.html
